### PR TITLE
Adding M.O.U.S. 9 Mouse MadCatz

### PR DESCRIPTION
**Pull request by FiloSpaTeam**

### CODE REVIEW

File: ratctl.py [hunk 1]
```diff
@@ -34,9 +34,9 @@
 # Mad Catz
             {'vid': 0x0738, 'pid': 0x1705, 'name': 'R.A.T 5'},  # @np's mouse (fork)
             {'vid': 0x0738, 'pid': 0x1709, 'name': 'R.A.T 9', 'battery': True},  # @improti's mouse (fork)
+            {'vid': 0x0738, 'pid': 0x1715, 'name': 'M.O.U.S. 9', 'battery': True},
             {'vid': 0x0738, 'pid': 0x1718, 'name': 'R.A.T PRO X'}]  # @Angelus's mouse (#8 - not tested)
 
-
```
Comment (MayeulC) [inline ratctl.py:39]: Doesn't it looks nicer with this new line? It's pretty subjective, though.

**Review by MayeulC — COMMENTED** [review]
Hi, and thanks a lot for opening this PR! Unfortunately,  I couldn't really have a look at it until now. Could you comment a bit on what are the capabilities of that mouse, though?

Does it have different DPI modes like the others? Is it possible to change it on the fly with a mouse button? Does the tool only work when the USB connector is wired in? (I would expect it to be the case). Does battery reporting works as expected? (I expect so). Mode changing (if applicable) when changing tabs, and changing mode changes tabs?

It would be great if you could quickly summarize that in your commit message :smile: 

Thanks again for your PR, very much appreciated :+1:

**Comment by FiloSpaTeam** [comment]
Oh yes, sorry for the new line. 
About the mouse i have a problem. I didnt bought, it was a present of a friend because he broke the usb dongle. I hacked the mouse using a bluetooth 4.0 receiver using bluetoothctl and i cannot read anything else at the moment.

The mouse is a long range wireless mouse that works on glass too. It has a precision button that can be programmable. It has a DPI switcher (i think, it s a button between left and right click button)

**Comment by MayeulC** [comment]
Well, then I have a few other questions :wink: 
* Does anything work at all with `ratctl`? If you couldn't test it, I would prefer not merging it into master (it could stay on a branch) until someone says it works. There are other untested devices in the code, but they all share very similar hardware.
* Is the USB ID the one of the dongle that doesn't work, or did you obtain it by another mean?

**Comment by FiloSpaTeam** [comment]
I read all from the mouse directly using bluetoothctl, i don t have the USB dongle, it s broken :) 
About ratctl it doesnt work for me, i think due to USB dongle that don t register the mouse under udev or lsusb but as bluetooth external.
I pulled to try with a working MOUS 9 ;)

**Comment by MayeulC** [comment]
Well, I'm not even sure there are settings that could be changed at all with `ratctl`, or if anything is configurable at all on this mouse :confused:

**Comment by FiloSpaTeam** [comment]
Maybe DPI but i cannot try without bluetoothctl lol

**Pull request closed without merging**